RocketChat/Rocket.Chat
# [FIX] Emoji not rendered on attachments description

## Proposed changes (including videos or screenshots)





## Issue(s)


## Steps to test or reproduce

Upload a file and add an emoji in the description (i.e `:eyes:`)
Should render emoji.

Attachment types influence the render. Attachments that should be tested:
Quotes (quotes are regarded as attachments)
TXT file
Image
Video
Audio

Custom attachments (the ones sent via API/Bots) like this one:
```
"message": {
        "rid": "{{rid}}",
        "msg": "Usando API",
        "attachments": [
            {
                "color": "#ff0000",
                "text": "Yay for gruggy! :scotland:"
            }
        ]
    }
```
## Further comments
